koku/api/report/queries.py

```diff
@@ -55,7 +55,7 @@ class ProviderMap(object):
 
     # main mapping data structure
     # this data should be considered static and read-only.
-    mapping = [
+    MAPPING = [
```

Comment:
This is fixing a convention error. When I created the map, this should have been all-caps to denote a constant that shouldn't be modified.